src/cache/cache_las.c

```diff
@@ -549,23 +549,26 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
     WT_MODIFY_VECTOR modifies;
     WT_SAVE_UPD *list;
     WT_SESSION_IMPL *session;
+    WT_TXN *txn;
     WT_TXN_ISOLATION saved_isolation;
-    WT_UPDATE *upd, *prev_upd;
-    wt_off_t las_size;
+    WT_UPDATE *prev_upd, *upd;
     WT_TIME_PAIR stop_ts_pair;
+    wt_off_t las_size;
+    wt_timestamp_t durable_timestamp;
     uint64_t insert_cnt, max_las_size;
     uint32_t mementos_cnt, btree_id, i;
-    uint8_t *p;
+    uint8_t *p, prepare_state, upd_type;
     int nentries;
-    bool las_key_saved, local_txn, squashed;
+    bool las_key_saved, local_txn, retrieve_modify, squashed;
```

Comment:
retrieve_modify may be redundant if we move the place we retrieve value from las